### PR TITLE
Update README with new python and botocore requirements

### PR DESCRIPTION
**Pull request by jillr**
##### SUMMARY
Update README with new python and botocore requirements

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,9 @@ PEP440 is the schema used to describe the versions of Ansible.
 
 ## Python version compatibility
 
-This collection depends on the AWS SDK for Python (Boto3 and Botocore). As AWS has [ceased supporting Python 2.6](https://aws.amazon.com/blogs/developer/deprecation-of-python-2-6-and-python-3-3-in-botocore-boto3-and-the-aws-cli/), this collection requires Python 2.7 or greater.
+This collection depends on the AWS SDK for Python (Boto3 and Botocore). Starting with the 2.0 releases of both collections, it is generally the collection's policy to support the versions of `botocore` and `boto3` that were released 12 months prior to the most recent major collection release, following semantic versioning (for example, 2.0.0, 3.0.0).
+
+As AWS has [ceased supporting Python 2.7](https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-python-2-7-in-aws-sdk-for-python-and-aws-cli-v1/), this collection requires Python 3.6 or greater.
```
Comment (tremble) [inline README.md:20]:
```suggestion
As the AWS SDK for Python (Boto3 and Botocore). has [ceased supporting Python 2.7](https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-python-2-7-in-aws-sdk-for-python-and-aws-cli-v1/), this collection requires Python 3.6 or greater.

Starting with the 2.0.0 releases of both collections, it is generally the collection's policy to support the versions of `botocore` and `boto3` that were released 12 months prior to the most recent major collection release, following semantic versioning (for example, 2.0.0, 3.0.0).

Version 2.0.0 of this collection supports `boto3 >= 1.13.0` and `botocore >= 1.16.0`

Comment (tremble) [reply README.md:20]: It feels a little odd specifying the modules we need at the start of the "Python version" section before defining which Python version we require.

**Review by tremble — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -15,7 +15,11 @@ PEP440 is the schema used to describe the versions of Ansible.
 
 ## Python version compatibility
 
-This collection depends on the AWS SDK for Python (Boto3 and Botocore). As AWS has [ceased supporting Python 2.6](https://aws.amazon.com/blogs/developer/deprecation-of-python-2-6-and-python-3-3-in-botocore-boto3-and-the-aws-cli/), this collection requires Python 2.7 or greater.
+As the AWS SDK for Python (Boto3 and Botocore). has [ceased supporting Python 2.7](https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-python-2-7-in-aws-sdk-for-python-and-aws-cli-v1/), this collection requires Python 3.6 or greater.
```
Comment (tremble) [inline README.md:18]:
```suggestion
As the AWS SDK for Python (Boto3 and Botocore) has [ceased supporting Python 2.7](https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-python-2-7-in-aws-sdk-for-python-and-aws-cli-v1/), this collection requires Python 3.6 or greater.
```

**Review by tremble — APPROVED** [review]
I've removed what looked like a stray full-stop (probably my fault).  Otherwise LGTM

**Review by Akasurde — APPROVED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -15,7 +15,11 @@ PEP440 is the schema used to describe the versions of Ansible.
 
 ## Python version compatibility
 
-This collection depends on the AWS SDK for Python (Boto3 and Botocore). As AWS has [ceased supporting Python 2.6](https://aws.amazon.com/blogs/developer/deprecation-of-python-2-6-and-python-3-3-in-botocore-boto3-and-the-aws-cli/), this collection requires Python 2.7 or greater.
+As the AWS SDK for Python (Boto3 and Botocore) has [ceased supporting Python 2.7](https://aws.amazon.com/blogs/developer/announcing-end-of-support-for-python-2-7-in-aws-sdk-for-python-and-aws-cli-v1/), this collection requires Python 3.6 or greater.
+
+Starting with the 2.0.0 releases of both collections, it is generally the collection's policy to support the versions of `botocore` and `boto3` that were released 12 months prior to the most recent major collection release, following semantic versioning (for example, 2.0.0, 3.0.0).
```
Comment (felixfontein) [inline README.md:20]: Is it always clear to readers what 'both' collections are? If not:
```suggestion
Starting with the 2.0.0 releases of amazon.aws and community.aws, it is generally the collection's policy to support the versions of `botocore` and `boto3` that were released 12 months prior to the most recent major collection release, following semantic versioning (for example, 2.0.0, 3.0.0).
```

Comment (Akasurde) [reply README.md:20]: I guess so.

Comment (Akasurde) [reply README.md:20]: Mentioning it explicitly won't hurt.

**Pull request merged**